css/navbar.css

```diff
@@ -65,6 +65,16 @@ nav li a:hover {
   margin: 25px;
 }
 
+.user-greet a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.user-greet a:hover {
+  color: #49a82e;
```

Comment:
Why are we changing the color on hover ??

Reply:
The other buttons on navbar seem to have the same property, that's why I had put `hover` property there.

Reply:
No for this button we don't need any color change, only cursor changing to `pointer` will do the job.